streamly-bytestring.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:  1.12
 name:           streamly-bytestring
-version:        0.1.4
+version:        0.1.5
```

Comment:
A version bump is not necessary

Reply:
Technically no, but you need one if you want to republish. I removed that change to let you handle this :)